docs/how-to/upgrade-cloud.rst

```diff
@@ -2,6 +2,103 @@
 Upgrade a cloud
 ===============
 
+Recommendations for upgrading production clouds
+-----------------------------------------------
+
+1. Be sure to upgrade in a maintenance window
+2. Start with the upgrade on control-plane applications
+3. Don't force the upgrade on non-empty hypervisors
+4. After upgrading the control-plane, choose one empty hypervisor to be a `canary node`.
+    After upgrading it, test if it's behaving as expected.
+5. If no issues are found after upgrading the `canary node`, proceed with the upgrade.
+
+Run upgrade for the whole cloud
+-------------------------------
+
+To upgrade the whole cloud in a single command.
+
+.. code:: bash
+
+    cou upgrade
+
+
+Run upgrade for the control-plane
+---------------------------------
+
+To upgrade a plan targeting the **control-plane** applications use:
+
+.. code:: bash
+
+    cou upgrade control-plane
+
+**Note:** Control-plane applications will be upgraded using the `all-in-one`_ method,
+meaning that all units will be upgraded at once.
+
+
+Run upgrade for the data-plane
+------------------------------
+
+To upgrade the **data-plane** applications use:
+
+.. code:: bash
+
+    cou plan data-plane
+
+**Note:**
+- It's essential to complete the upgrade of the **control-plane** components before
+being able to upgrade for **data-plane**.
+- Nova-compute and services colocated on the same machines will be upgraded using the
+`paused-single-unit`_ method, meaning that it will upgrade unit by unit.
```

Comment:
Also this. It would be better suited in `explanation` which I'm adding in my upcoming patch.